appveyor.yml

```diff
@@ -10,22 +10,15 @@ environment:
     secure: PbT7/2muaC99K24vd6d+DMQETnal230eqmdPtw4tn+zqhwkKpSRIJgjRmJPxphBF
 install:
 - cmd: >-
-    choco install ninja -y
-
-    choco source add -n=AFG -s="https://api.bintray.com/nuget/anotherfoxguy/choco-pkg"
-
-    choco install conan -y
-
-    refreshenv
-
+    choco source add -n=AFG -s="https://api.bintray.com/nuget/anotherfoxguy/choco-pkg"

+    choco install conan -y

+    refreshenv

     git submodule update --init --recursive
 
-    call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat"
 build_script:
 - cmd: >-
-    cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DUSE_PMM=ON .
-
-    ninja
+    cmake . -G "Visual Studio 15 2017" -DCMAKE_BUILD_TYPE=Release -A Win32 -DCMAKE_BUILD_TYPE=Release


```

Comment:
Remove duplicate `DCMAKE_BUILD_TYPE`

Reply:
What does ^M mean? Is it supposed to be there?